### PR TITLE
GetHighScore method

### PR DESCRIPTION
**Pull request by marcelkuipers**
This PR implements the getHighScore method in the scoreController. It also fixes a bug with regards to reading and writing scores.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/score/ScoreController.java [hunk 1]
```diff
@@ -47,5 +49,20 @@ public void addScore(Score score) {
         }
     }
 
+    /**
+     * Return the high score.
+     * @return the high score
+     */
+    public int getHighScore() {
+        System.out.println(Arrays.toString(scores.toArray()));
+        int highest = 0;
```
Comment (NilsHullegien) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/score/ScoreController.java:56]: Please remove this print.

**Comment by NilsHullegien** [comment]
Besides my minor comment, I approve of this code. Of course once it is merged into master it will be combined with the saving of the score.

**Comment by marcelkuipers** [comment]
Oops, fixed that now.

**Comment by NilsHullegien** [comment]
Looks good now, I approve!

**Comment by dVoorhout** [comment]
Looks good to me

**Pull request merged**